### PR TITLE
docs(usage): Update usage documentation

### LINKED ISSUE #15: Add adapter locally on the repo and update instructions

### PR DESCRIPTION
**Pull request by raisedadead**
Closes #15

### CODE REVIEW

**Comment by BerkeleyTrue** [comment]
We should provide instructions for those who want to install it globally for all repos

**Comment by raisedadead** [comment]
@BerkeleyTrue I have updated the instructions. Let me know if these are ok, thanks.

File: README.md [hunk 1]
```diff
@@ -20,31 +20,39 @@ This Commitizen adapter helps standardize commit messages for freeCodeCamp so
...
-# Install freeCodeCamp specific adapter
-git clone https://github.com/freeCodeCamp/cz-freecodecamp.git
+# Install the freeCodeCamp commitizen adapter globally
+npm install --global cz-freecodecamp
 
-# Add freeCodeCamp plugin to be used
-cd cz-freecodecamp
-echo '{ "path": "'$(pwd)'" }' > ~/.czrc
+# Save the adapter into global config
+echo '{ "path": "cz-freecodecamp" }' > ~/.czrc
```
Comment (BerkeleyTrue) [inline README.md:35]: This won't work with global installs. The entire path of the global installs is needed.

For me it's `{ "path": "/usr/local/lib/node_modules/cz-conventional-changelog" }`
But this is going to be specific to the user. We need a way to programmatically get the users global npm install path

**Review by BerkeleyTrue — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (erictleung) [reply README.md:35]: I haven't been able to test this locally yet, but according to their docs, that *should* work.

Source: https://github.com/commitizen/cz-cli#conventional-commit-messages-as-a-global-utility

Comment (BerkeleyTrue) [reply README.md:35]: Ok, it works! It didn't work for me before. Thanks @erictleung

**Review by BerkeleyTrue — APPROVED** [review]

**Comment by raisedadead** [comment]
Great that this worked out! Thanks for the QA.

**Pull request merged**